saleor/settings.py

```diff
@@ -305,9 +307,21 @@
             r'.+\.hot-update\.js',
             r'.+\.map']}}
 
+ENABLE_ELASTICSEARCH = ast.literal_eval(os.environ.get('ENABLE_ELASTICSEARCH', 'False'))
 
-BOOTSTRAP3 = {
-    'set_placeholder': False,
-    'success_css_class': '',
-
-}
+if ENABLE_ELASTICSEARCH:
+    HAYSTACK_CONNECTIONS = {
+        'default': {
+            'ENGINE': 'haystack.backends.elasticsearch_backend.ElasticsearchSearchEngine',
+            'URL': os.environ.get('ELASTICSEARCH_URL',
```

Comment:
We need to also handle `BONSAI_URL` and `SEARCHBOX_URL` (these are Heroku add-ons that provide Elasticsearch).